nixos/modules/programs/wayland/hyprland.nix

```diff
@@ -29,14 +28,14 @@ in
 
     finalPackage = mkOption {
       type = types.package;
-      readOnly = true;
```

Comment:
Why remove this?

Reply:
I'm not sure where this would be useful, considering the Nixpkgs derivation is updated based on the flake one, and both have those flags.

Reply:
Not anymore, the `hidpiXWayland` flag was just removed and `nvidiaPatches` renamed to `enableNvidiaPatches`.

In the current state, the Hyprland module is only guaranteed to be usable with the Hyprland package from the same branch. For example, because of the recent flag updates, we can't use the 23.05 module with the package from the unstable channel anymore.

This would however be possible if we allowed users to set directly the `finalPackage` option.